tests/avocado/test_nec-usb-xhci.py

```diff
@@ -0,0 +1,53 @@
+import copy
+import os
+
+from avocado_qemu import test
+from avocado.utils import process
+
+class TestNecUsbXhci(test.QemuTest):
+    """
+    Run with:
+
+        avocado run test_nec-usb-xhci.py \
+        -m test_nec-usb-xhci.py.data/parameters.yaml
+
+    :avocado: enable
+    :avocado: tags=usbstorage
+    """
+
+    def setUp(self):
+        usbdevice = os.path.join(self.workdir, 'usb.img')
+        process.run('qemu-img create -f qcow2 %s 10M' % usbdevice)
```

Comment:
well `dd` created `raw` image would be probably simpler... Anyway did you consider looking for `qemu-img` binary? It might not be installed system-wide and only available from sources....